GA-worksheet.md

```diff
@@ -26,44 +31,58 @@
 ##Project #2 Part #2: Style the Header with CSS
 
 1. What advantages does the `<link>` tag have over the `<style>` tag?
+    You can its better in an orginization sense.
 
 2. What is normalize.css?
+    The CSS File
 
 3. Why do we need to add a padding property to the `<ul>` tag?
+    To add space between the links / text
```

Comment:
We added padding to the individual <li> tags to add space around the links.

The  padding around the `<ul>` is tricky --  there is default padding on the left for unordered lists to accommodate the bullet points, so adding padding to all sides makes this particular ul appear centered.